default.nix

```diff
@@ -0,0 +1,49 @@
+# Default import pinned pkgs
+{ pkgsSrc ? (import ./nix/pkgs.nix {}).pkgsSrc
+, pkgs ? (import ./nix/pkgs.nix { inherit pkgsSrc; }).pkgs
+, doCheck ? true
+}: with pkgs;
+
+let
+  # Get contract dependencies from lock file
+  inherit (callPackage ./nix/dapp.nix {}) specs packageSpecs;
+  inherit (specs.this) deps;
+
+  # Import deploy scripts from dss-deploy
+  dss-deploy = import deps.dss-deploy.src' {};
+
+  # Create derivations from lock file data
+  packages = packageSpecs (deps // {
+    # Set specific solc versions for some contract derivations
+    multicall = deps.multicall // { solc = solc-versions.solc_0_4_25; };
+    vote-proxy = deps.vote-proxy // { solc = solc-versions.solc_0_4_25; };
+  });
+in makerScriptPackage {
```

Comment:
Where is `makerScriptPackage` defined?

Reply:
Also in the pinning repo: https://github.com/makerdao/nixpkgs-pin/blob/master/maker.nix#L15

Reply:
what do you think about something like `mkDappDerivation`? It's not really MakerDAO-specific right?

Reply:
Hmm looked at it, it is specific to the tdds deployment scripts somehow. I'd still keep the `mk` prefix though.

Reply:
I guess it's not, but didn't want to be too general in this first iteration to keep a smaller scope.

Can definitely add this to improvements, and even to `dapp2nix` maybe?

Essentially the function is for packaging scripts that are using `dapp`.

Reply:
Let's keep this in mind for a next iteration then.